client/src/client-tests/index.test.js

```diff
@@ -9,7 +9,7 @@
 
 // test('clicking the magic button changes its text based on the state', () => {
 
-//   render(<App />);
+//   render(<A pp />);
```

Comment:
typo on App? Just wanted to bring it up in case you decide to uncomment this out in the future